frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.test.tsx

```diff
@@ -74,13 +73,13 @@ const mockClusterCurator: ClusterCurator = {
     spec: {
         install: {
             towerAuthSecret: 'ansible-test-secret',
-            prehook: [{ name: 'test-job-pre-install', extra_vars: {} }],
-            posthook: [{ name: 'test-job-post-install', extra_vars: {} }],
```

Comment:
these are still not unique between stages.  We has a bug on this at one point and the job names should be unique across stages.  i.e. spec.install.prehook[0].name should not equal spec.upgrade.prehook[0].name